### DISCUSSION TITLE
Create Recipe by Copy Paste HTML

### DISCUSSION THREAD

**Discussion by coolkau** [mealie-recipes/mealie#1879] [Feature Request]
For instance the Guardian has several recipes on one homepage. Mealie cannot handle this. 
I would be cool if I could copy/paste the relevant section into a field and Mealie runs the normal scraper over just the copied section.

**Comment by michael-genson** [comment]
More discussion on importing via copy/pasted HTML here: https://github.com/mealie-recipes/mealie/discussions/2391
